dts/bindings/pinctrl/openisa,rv32m1-pinctrl.yaml

```diff
@@ -0,0 +1,68 @@
+# Copyright (c) 2022 Henrik Brix Andersen <henrik@brixandersen.dk>
+# Copyright (c) 2022, NXP
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  OpenISA RV32M1 pinctrl node. This node will define pin configurations in pin groups,
+  and has the 'pinctrl' node identifier in the SOC's devicetree. Each group
+  within the pin configuration defines the pin configuration for a peripheral,
+  and each numbered subgroup in the pin group defines all the pins for that
+  peripheral with the same configuration properties. The 'pins' property in
+  a group selects the pins to be configured, and the remaining properties set
+  configuration values for those pins. Here is an example group for UART0 pins:
+
+  uart0_default: uart0_default {
+    group0 {
+      pins = <LPUART0_RX_PTC7
+        LPUART0_TX_PTC8>;
+      drive-strength = "low";
+      slew-rate = "fast";
```

Comment:
nit: `pins` -> `pinmux`, alignment should be fixed as well

Reply:
Fixed.